index.html

```diff
@@ -203,7 +258,8 @@ <h2 class="description-2">I'm always interested in hearing about new projects,
                                 <input type="email" name="email-input" id="email-input" placeholder="    Email address" required>
                             </label><br>
                             <label for="more-text">
-                                <textarea name="" id="more-text" cols="30" rows="6" maxlength="500" placeholder="Enter text here" required></textarea>
+                                <textarea name="" id="more-text" cols="30" rows="6" maxlength="500" placeholder="Enter text here" class="mob" required></textarea>
+                                <textarea name="" id="more-text" cols="77" rows="6" maxlength="500" placeholder="Enter text here" class="desk" required></textarea>
                             </label>
```

Comment:
- [OPTIONAL] I think would be better to make sure the text area is aligned with the other inputs as in the Figma design

__After checking out the commit before the previous review I noticed that the errors were there before, that's why it's optional for now.__

- Your design:
![Screenshot from 2023-01-21 15-31-30](https://user-images.githubusercontent.com/97834160/213869126-55bcb681-8bf8-4a80-a3f8-d9ba27d6bf3c.png)


- Figma:

![Screenshot from 2023-01-21 14-40-27](https://user-images.githubusercontent.com/97834160/213869082-d8db67ca-7ddb-4065-9b4c-c8b8d9d53e94.png)